src/delegation/store/memory.rs

```diff
@@ -258,17 +257,19 @@ where
                             continue;
                         }
 
-                        for target_pred in policy.iter() {
-                            for delegate_pred in delegation.payload.policy.iter() {
-                                let comparison =
-                                    target_pred.harmonize(delegate_pred, vec![], vec![]);
+                        // FIXME
```

Comment:
Sitting in a commented-out `FIXME` until we write the predicate subsumption logic (which I'll switch back to after fixing a couple other things that I'm unblocking)